package.json

```diff
@@ -3,21 +3,45 @@
   "version": "1.0.0-alpha.6",
   "private": "true",
   "scripts": {
-    "build": "node index.js",
-    "commit": "cz"
+    "build": "pnpm build:tokens && pnpm build:resets && pnpm build:cc",
+    "commit": "cz",
+    "semantic-release": "semantic-release",
+    "build:cc": "cd component-classes && tsc && vite build",
+    "build:tokens": "cd tokens && node index.js",
+    "build:resets": "cd resets && node index.js"
   },
   "type": "module",
+  "exports": {
+    "./component-classes": "./component-classes/index.js",
+    "./components": "./dist/components.css",
+    "./component-classes/classes": "./component-classes/classes.js",
+    "./component-classes/shortcuts": "./component-classes/shortcuts.js"
+  },
+  "types": "./component-classes/index.d.ts",
+  "files": [
+    "./component-classes/index.js",
+    "./component-classes/classes.js",
+    "./component-classes/shortcuts.js",
+    "./component-classes/index.d.ts"
+  ],
```

Comment:
In [component-classes repo](https://github.com/warp-ds/component-classes/blob/alpha/package.json#L16) we also had `dist/assets` folder here. Just checking if it's no longer something we need to publish to npm.

Reply:
Yea, I figured that we probably do not need to add it to npm as the EIK version of the css is supposed to be used.
I can't even see it here https://www.npmjs.com/package/@warp-ds/component-classes?activeTab=code 🤔 Maybe it never worked. Or maybe I'm missing something here 😅

Reply:
Sounds right!